main_test.go

```diff
@@ -361,6 +362,43 @@ func TestAlises(t *testing.T) {
 	}
 }
 
+func TestParseIniFile(t *testing.T) {
+	configHomeEnv := "XDG_CONFIG_HOME"
+	originalConfigHome := os.Getenv(configHomeEnv)
+	defer os.Setenv(configHomeEnv, originalConfigHome)
+
+	tests := []struct {
+		message       string
+		configPath    string
+		expectedError string
+	}{
+		{
+			message:    "valid ini file should parse correctly",
+			configPath: path.Join("testdata", "ini", "valid"),
+		},
+		{
+			message:    "absent ini file should be ignored",
+			configPath: path.Join("testdata", "ini", "missing"),
+		},
+		{
+			message:       "invalid ini file should raise error",
+			configPath:    path.Join("testdata", "ini", "invalid"),
```

Comment:
nit: make `configPath` just have the "valid" or "invalid" etc, and have the test runner set the rest of the path